packages/slate-editor/src/modules/editor-v4-floating-add-menu/FloatingAddMenu.tsx

```diff
@@ -164,7 +165,7 @@ export function FloatingAddMenu<Action>({
                     />
                 )}
             </TooltipV2.Tooltip>
-            {!open && (
+            {!open && !hasOnlySpaces(text) && (
```

Comment:
@OskarGasiorowski @aspirisen 

Do you guys think we should additionally disable `/` hotkey reaction when the placeholder is not visible?

![Peek 2022-04-27 14-48](https://user-images.githubusercontent.com/370680/165512017-3a03a38f-2f2b-4f60-9e29-ab1900d820e3.gif)

Reply:
So basically `/` hotkey should work only if a line is empty?

Reply:
Maybe, but I'm not sure yet.

@aspirisen what do you think about it?

Reply:
hm, personally i am not against current behaviour, i would leave it, at least for now